tests/test_trace_frame.py

```diff
@@ -9,6 +9,11 @@ def test_trace_frame_validation_passes(trace_frame_data):
     assert frame
 
 
+def test_trace_no_memory():
+    raw_frame = {"pc": 0, "op": "PUSH1", "gas": 4732305, "gasCost": 3, "depth": 1, "stack": []}
```

Comment:
I get this trace from calling this method: https://github.com/fubuloubu/ERC4626/blob/main/contracts/test/Token.vy#L68-L70

which must not use any memory?

Reply:
I should add that I was using `geth` and not `hardhat` when doing this.

Reply:
> which must not use any memory?

It might not be